pkg/store/store.go

```diff
@@ -130,12 +140,18 @@ func (store *OpenTSDBStore) Series(
 	})
 	if err != nil {
 		level.Error(store.logger).Log("err", err)
+		if tr, ok := trace.FromContext(ctx); ok {
```

Comment:
nit: Could use new `ctx` variable on line 138 (github doesn't let you put comments more than 3 lines away from changed code, sigh).